### DISCUSSION TITLE
Request: Audio Meters

### DISCUSSION THREAD

**Discussion by ready-1** [videovillage/community#129] [Ideas]
A set of audio meters would be incredibly useful.  Something along the lines of what you find on most decent monitors.  Multi-channel would be nice.  Need to be able to see how many channels are there and the levels in each.

**Comment by slgh-hh** [comment]
I  would appreciate this, too!! Would turn the best macos pro player into an even better pro player!!

**Comment by stanpete78** [comment]
+1 Screen is great but we really need multichannel audio with selectable tracks.
